packages/react-storage/src/components/StorageBrowser/context/actions/listLocationItemsAction.ts

```diff
@@ -95,12 +95,16 @@ export async function listLocationItemsAction(
     },
   };
 
-  const output = await list(listInput);
+  try {
+    const output = await list(listInput);
 
-  const result = [
-    ...(refresh ? [] : prevState.result),
-    ...parseResult(output, path),
-  ];
+    const result = [
+      ...(refresh ? [] : prevState.result),
+      ...parseResult(output, path),
+    ];
 
-  return { result, nextToken: output.nextToken };
+    return { result, nextToken: output.nextToken };
+  } catch (e) {
+    return { result: [], nextToken: undefined, message: (e as Error).message };
+  }
```

Comment:
Error `message` is already captured and can be read from:
```ts
const [{ message }] = useAction({ type: 'LIST_LOCATION_ITEMS; });
```

Reply:
Thanks, updated in latest